docs/source/howto/codes.rst

```diff
@@ -45,8 +45,92 @@ Design guidelines
 Running external codes
 ======================
 
-`#3987`_
+The process of running an external code through AiiDA involves loading and instantiating the appropriate ``CalcJob`` wrapping class for the code, filling the inputs necessary, and then running or submitting the constructed object.
 
+If the ``CalcJob`` has been registered in AiiDA (see section :ref:`on how to register plugins<how-to:plugins:entrypoints>`) you can use the ``CalculationFactory``:
+
+.. code-block:: python
+
+    from aiida.plugins import CalculationFactory
+    calculation_builder = CalculationFactory('arithmetic.add').get_builder()
```

Comment:
```suggestion
    from aiida.plugins import CalculationFactory
    calculation_class = CalculationFactory('arithmetic.add')
```
I would assign the class explicitly to make it clear that that is what the factory returns.
I would also not use the builder straight away but go for the simple dictionary syntax